### PR TITLE
docs: allow running tutorial from docker 

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: docs/source/concepts/deploy.rst [hunk 1]
```diff
@@ -49,7 +49,18 @@ Run :code:`bentoml containerize` to start the containerization process.
     INFO [cli] Successfully built docker image "iris_classifier:ejwnswg5kw6qnuqj"
 
 
-Built Docker images are stored to the local Docker repository.
+.. dropdown:: For Mac with Apple Chip
```
Comment (aarnphm) [inline docs/source/concepts/deploy.rst:52]:
```suggestion
.. dropdown:: For Mac with Apple Silicon
```

File: docs/source/concepts/deploy.rst [hunk 2]
```diff
@@ -49,7 +49,18 @@ Run :code:`bentoml containerize` to start the containerization process.
...
+   :icon: cpu
+
+   Specify the :code:`--platform` to avoid potential compatibility issues with some
+   Python libraries.
+
+   .. code:: bash
+
+      bentoml containerize iris_classifier:latest --platform=linux/amd64
```
Comment (aarnphm) [inline docs/source/concepts/deploy.rst:60]: Just a preference
```suggestion
      bentoml containerize --platform=linux/amd64 iris_classifier:latest
```

File: docs/source/tutorial.rst [hunk 3]
```diff
@@ -33,46 +33,50 @@ container image for deployment.
 Setup for the tutorial
 ----------------------
 
-There are two ways to complete this tutorial: you can either run the code in browser
+There are 3 ways to complete this tutorial: you can either run the code in browser
```
Comment (aarnphm) [inline docs/source/tutorial.rst:36]: grammar stuff ;(

```suggestion
There are three ways to complete this tutorial: you can either run the code in browser
```

File: docs/source/tutorial.rst [hunk 4]
```diff
@@ -381,6 +384,16 @@ via the :code:`bentoml containerize` CLI command:
    You will need to `install Docker <https://docs.docker.com/get-docker/>`_ before
    running this command.
 
+.. dropdown:: For Mac with Apple Chip
```
Comment (aarnphm) [inline docs/source/tutorial.rst:387]:
```suggestion
.. dropdown:: For Mac with Apple Silicon
```

**Review by aarnphm — COMMENTED** [review]
LGTM with some grammatical changes

File: docs/source/tutorial.rst [hunk 5]
```diff
@@ -381,6 +384,16 @@ via the :code:`bentoml containerize` CLI command:
...
+   :icon: cpu
+
+   Specify the :code:`--platform` to avoid potential compatibility issues with some
+   Python libraries.
+
+   .. code:: bash
+
+      bentoml containerize iris_classifier:latest --platform=linux/amd64
```
Comment (parano) [inline docs/source/tutorial.rst:395]:
```suggestion
      bentoml containerize --platform=linux/amd64 iris_classifier:latest
```

File: docs/source/tutorial.rst [hunk 6]
```diff
@@ -33,8 +33,7 @@ container image for deployment.
 Setup for the tutorial
 ----------------------
 
-There are three ways to complete this tutorial: you can either run the code in browser
-with Google Colab, or you can set up a local development environment on your computer.
+There are three ways to complete this tutorial:
```
Comment (aarnphm) [inline docs/source/tutorial.rst:36]: request: should we make a table and hyperlink each section to the section below?

Comment (parano) [reply docs/source/tutorial.rst:36]: the first two parts are very short that fits in one page, probably not worth adding another hyperlink section here

**Pull request merged**